src/EditorFeatures/Test/Diagnostics/IDEDiagnosticIDConfigurationTests.cs

```diff
@@ -87,6 +87,12 @@ private static void ValidateHelpLinkForDiagnostic(string diagnosticId, string he
             return;
         }
 
+        if (diagnosticId == "FileBasedPrograms")
```

Comment:
Can we use the `FileLevelDirectiveDiagnosticAnalyzer.DiagnosticId` constant here?